### PR TITLE
feat: add mastodon icon [WPDS-1291]

### PR DESCRIPTION
**Pull request by atomictangerine**
Adds mastodon icon

### CODE REVIEW

**Comment by artmsilva** [comment]
I left a note on the preview.

File: src/mastadon.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" viewbox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (artmsilva) [inline src/mastadon.svg:1]: We will want to remove the width and height attributes. We use viewBox.

**Review by artmsilva — COMMENTED** [review]
The svg should be named "mastodon". https://en.wikipedia.org/wiki/Mastodon_(social_network)

**Comment by atomictangerine** [comment]
Thanks for catching that typo

**Review by artmsilva — APPROVED** [review]

**Pull request merged**